test/model.test.js

```diff
@@ -4821,7 +4821,7 @@ describe('Model', function() {
         });
       });
 
-      it('watch() (gh-5964)', function() {
+      it.skip('watch() (gh-5964)', function() {
```

Comment:
intentionally skipped?

Reply:
Yes for now. We don't test against replica sets on travis currently, that's a separate concern, so I added this skip in https://github.com/Automattic/mongoose/pull/6363/commits/b44367885fa4168db58b09f4d76b4a08dc0abdbd

Reply:
:+1: